### ISSUE TITLE
"Duplicate" button should perform deep copy 

### ISSUE THREAD

**Issue by jopasserat** [openmole/openmole#292]
When clicking on "Duplicate" on a file in the workspace, OM performs a shallow copy: a symlink is created 

In the example below I want to duplicate the file `hypertune_GA.oms` to a new  `hypertune_GA_cifar100.oms`.

The initial file is a symlink pointing to another location (part of a different set of files versioned under git).

When clicking duplicate in the GUI, a new link pointing to the same file is created... so editing the "duplicated" file ends up modifying the initial file!

```
jpassera@hornbeam:/vol/biomedic2/jpassera/diana/hypertune$ pushd ~/.openmole/bardolph.doc.ic.ac.uk/webui/projects/testGAs/
~/.openmole/bardolph.doc.ic.ac.uk/webui/projects/testGAs /vol/biomedic2/jpassera/diana/hypertune ~
jpassera@hornbeam:~/.openmole/bardolph.doc.ic.ac.uk/webui/projects/testGAs$ ls -lh
total 52K
drwxr-xr-x 2 jpassera vip 8.0K May  1 22:16 evolution
drwxr-xr-x 2 jpassera vip 4.0K May  1 22:14 evolution--cifar100
lrwxrwxrwx 1 jpassera vip   56 May  1 13:54 hypertune_GA_cifar100.oms -> /vol/biomedic2/jpassera/diana/hypertune/hypertune_GA.oms
lrwxrwxrwx 1 jpassera vip   56 Apr 30 19:53 hypertune_GA.oms -> /vol/biomedic2/jpassera/diana/hypertune/hypertune_GA.oms
```

**Comment by mathieuleclaire** [comment]
Is this not what NOFOLLOW_LINKS is about ? Cause it is the option used for this copy. Thus we could expect their content are independent after the copy !

https://github.com/openmole/openmole/blob/dev/openmole/gui/server/org.openmole.gui.server.core/src/main/scala/org/openmole/gui/server/core/Utils.scala#L178:L189
https://github.com/openmole/openmole/blob/dev/openmole/third-parties/org.openmole.tool.file/src/main/scala/org/openmole/tool/file/package.scala#L108

**Comment by jopasserat** [comment]
You're right. We should have a way to specify this behaviour as an option of `copy`.
Should default to what it's doing now, but with the possibility to override the copy flags.

**Comment by mathieuleclaire** [comment]
The fact is that NOFOLLOW_LINKS is the only enum of LinkfOption
https://docs.oracle.com/javase/7/docs/api/java/nio/file/LinkOption.html

**Comment by jopasserat** [comment]
Yes it should only be specified if you don't want the JRE to follow the symlink.
By default it does follow: https://stackoverflow.com/a/20052770/470341

**Comment by jopasserat** [comment]
No that's the thing (it's confusing), we use the option so we're not following the links. But I think we should follow them in this special case (at least that's the behaviour I expect to find as a user due to the initial issue of editing the same .oms file in the WebUI).

**Comment by jopasserat** [comment]
the issue is still present as of today

**Comment by romainreuillon** [comment]
Might be fixed by 09eb460f9165f8a47029eaa0601560c236fb678f

Guaranteed 100% not tested
